server/crashmanager/views.py

```diff
@@ -735,6 +735,40 @@ def createExternalBug(request, crashid):
         raise SuspiciousOperation
 
 
+def createExternalBugTemp(request, crashid):
+    entry = get_object_or_404(CrashEntry, pk=crashid)
+    check_authorized_for_crash_entry(request, entry)
+
+    if not entry.bucket:
+        return renderError(request, ("Cannot create an external bug for an issue that is not associated "
+                                     "to a bucket/signature"))
+
+    if request.method == 'GET':
+        if 'provider' in request.GET:
+            provider = get_object_or_404(BugProvider, pk=request.GET['provider'])
+        else:
+            user = User.get_or_create_restricted(request.user)[0]
+            provider = get_object_or_404(BugProvider, pk=user.defaultProviderId)
+
+        template = provider.getInstance().getTemplateForUser(request, entry)
+
+        # attachmentData = {}
+
+        # if template:
+        #     attachmentData = self.substituteTemplateForCrash(template, entry)
```

Comment:
Just curious - what's the reason for leaving these commented lines in place?

Reply:
I guess that was a way for me to remember that I will have to handle `attachmentData` coming from `subsituteTemplateForCrash` in the next PR (which will support the test case). That's cleaner to remove it though, will do in the next commit.